src/simulator.zig

```diff
@@ -30,10 +37,15 @@ pub const log_level: std.log.Level = if (log_state_transitions_only) .info else
 /// Modifies compile-time constants on "config.zig".
 pub const deployment_environment = .simulation;
 
+const accounts_batch_size_max = @divFloor(config.message_size_max - @sizeOf(vsr.Header), @sizeOf(tb.Account));
+const transfers_batch_size_max = @divFloor(config.message_size_max - @sizeOf(vsr.Header), @sizeOf(tb.Transfer));
```

Comment:
Since it needs to be comptime, the `StateMachine` would need access to the `message_body_size_max` as comptime (currently it only has it at runtime). What do you think of passing `StateMachine` a `comptime config: struct { message_body_size_max: usize }`. Then it could export these on its `Options` type as declarations, to avoid having two different things called `options`.